ibm-cloud-sdk.gemspec

```diff
@@ -11,7 +11,8 @@ Gem::Specification.new do |spec|
   spec.licenses      = %w["Apache-2.0"]
   spec.required_ruby_version = Gem::Requirement.new(">= 2.3.0")
 
-  spec.metadata["homepage_uri"] = spec.homepage
+  spec.metadata["homepage_uri"]  = spec.homepage
+  spec.metadata["changelog_uri"] = "https://github.com/IBM-Cloud/ibm-cloud-sdk-ruby/blob/master/CHANGELOG.md"
```

Comment:
I think the changelog URI needs to point to the current release. Looks like you can use `spec.version` to build the appropriate URI string. (I'm looking at [this example from the rspec gem](https://github.com/rspec/rspec-core/blob/main/rspec-core.gemspec#L18)).

Reply:
If you use `spec["changelog_uri"] = "https://github.com/IBM-Cloud/ibm-cloud-sdk-ruby/blob/v#{spec.version}/CHANGELOG.md"` then the URI will point to the changelog for the current tag (I think that's what we want on the RubyGems.org page, right?)